app/src/org/commcare/tasks/ResourceEngineTask.java

```diff
@@ -81,14 +81,21 @@ protected AppInstallStatus doTaskBackground(String... profileRefs) {
             AndroidCommCarePlatform platform = app.getCommCarePlatform();
             ResourceTable global = platform.getGlobalResourceTable();
 
+            if (reinstall) {

```

Comment:
Just wanted to confirm that there's no way to do this in a way that doesn't wreck a functional installation until it's done (by, say, staging the install into a new temporary sandbox with a new GUID or something similar)

Reply:
Hey, I thought we decided to do it in place since we assume that the current installation will anyway be non functional when we are triggering reinstall. 

We can definitely follow the update approach where we first do the install in another table and then move it to the global but not sure whether it's worth that complexity for the reinstall usecase.